appservice/src/utils/javaUtils.ts

```diff
@@ -12,8 +12,17 @@ export namespace javaUtils {
     const DEFAULT_PORT: string = '8080';
     const PORT_KEY: string = 'PORT';
 
-    export function isJavaTomcatRuntime(runtime: string | undefined): boolean {
-        return !!runtime && runtime.toLowerCase().startsWith('tomcat');
+    export function needDeployWarFile(runtime: string | undefined): boolean {
+        if (!runtime) {
```

Comment:
This can be done in one line with the use of a RegExp:
```
return !!runtime && /^(tomcat|wildfly)/i.test(runtime);
```

If you're not familiar with RegExp, you can use a site like this to get explanations and test it out: https://regex101.com/r/1oyXnJ/1